### PR TITLE
4

### PR DESCRIPTION
**Pull request by xyzze**

### CODE REVIEW

File: src/AtlasModules/atlas-config.bat [hunk 1]
```diff
@@ -328,20 +330,20 @@ for /f %%i in ('wmic path Win32_USBController get PNPDeviceID ^| findstr /L "PCI
...
+:: enable MSI mode on GPU
 for /f %%i in ('wmic path Win32_VideoController get PNPDeviceID ^| findstr /L "PCI\VEN_"') do (
     reg add "HKLM\System\CurrentControlSet\Enum\%%i\Device Parameters\Interrupt Management\MessageSignaledInterruptProperties" /v "MSISupported" /t REG_DWORD /d "1" /f
 )
 for /f %%i in ('wmic path Win32_VideoController get PNPDeviceID ^| findstr /L "PCI\VEN_"') do (
     reg add "HKLM\System\CurrentControlSet\Enum\%%i\Device Parameters\Interrupt Management\Affinity Policy" /v "DevicePriority" /f >nul 2>nul
 )
 
-:: enable MSI Mode on network adapters
+:: enable MSI node on network adapters
```
Comment (SyrmGIT) [inline src/AtlasModules/atlas-config.bat:341]: typo "node"

**Comment by xyzze** [comment]
fixed

**Review by SyrmGIT — APPROVED** [review]

**Comment by PencilNavigator** [comment]
consider changing the pull request title?

**Comment by xyzze** [comment]
> consider changing the pull request title?

yeah done

**Pull request merged**